File: lab_ma_flutter/lib/types/event.dart
import 'package:image_picker/image_picker.dart';
import 'package:uuid/uuid.dart';

class Event {
  final String id = const Uuid().v4();
  String name;
  XFile image;
  DateTime date;
  String location;
  bool isFavorite = false;
  final String username;
  List<String> tags;

  Event({
    required this.name,
    required this.image,
    required this.date,
    required this.location,
    required this.username,
    required this.tags
  });
}

File: lab-ma-memory/lab-ma/Types/Event.swift
//
//  Event.swift
//  lab-ma
//
//  Created by Dimitrie-Toma Furdui on 06.11.2022.
//

import Foundation

struct Event: Identifiable {
    var id: String = UUID().uuidString
    var name: String
    var date: Date
    var location: String
    var imageData: Data
    var isFavorite = false
    var tags: [String]
    var username: String
}
